src/core/mod.rs

```diff
@@ -105,6 +106,7 @@ pub const BITCOIN_MAINNET_FIRST_BLOCK_HASH: &str =
 pub const BITCOIN_MAINNET_INITIAL_REWARD_START_BLOCK: u64 = 651389;
 pub const BITCOIN_MAINNET_STACKS_2_05_BURN_HEIGHT: u64 = 713_000;
 pub const BITCOIN_MAINNET_STACKS_21_BURN_HEIGHT: u64 = 781_551;
+pub const BITCOIN_MAINNET_STACKS_22_BURN_HEIGHT: u64 = 787_700;
```

Comment:
The SIP mentions block 787551. Either seems okay to me, but we should make sure they match.

Reply:
Yeah, I agree. I'll leave this thread open / unresolved until we have a conclusion there. I just wanted to put down something approximating the latest possible block in the PR.